docs/src/showcase.md

```diff
@@ -40,6 +40,8 @@ title: "Community Showcase"
 * [Testim Playground](https://www.testim.io/playground/): Record Playwright UI tests as code
 * [Try Playwright](https://try.playwright.tech/): Interactive playground for Playwright to run examples directly from your browser
 * [Applitools](https://applitools.com): Add AI-powered visual assertions and run your Playwright tests on all browser, device, and viewport combinations in parallel, without requiring any setup.
+* [Sauce Labs](https://saucelabs.com/platform/automation-tools/playwright): Run Playwright tests in parallel in the cloud or your own docker environment.
```

Comment:
I believe the rest of the list except Applitools goes in alphabetic order, let's fix the exceptions

Reply:
👍 I'll re-align.

Reply:
I sorted the `Tools` section 👍 
I left the `Examples` section in its original order, since there are quite a few items that weren't alphabetically sorted. Unless you'd like me to sort those too?

Reply:
No, it's ok, thank you!